docs/community/dev-card.md

```diff
@@ -0,0 +1,16 @@
+---
+id: dev-card
+title: 'Showcase your open source stats with your dev card'
+sidebar_label: 'Dev Card'
+keywords:
+- dev-card 
+---
+
+The Dev card feature is what we like to call your "open source business card". At the front, you see your username, the amount of pull requests you create, repositories you contribute to, a `View Profile` button that takes you to your dashboard, and a green icon, which describes your activity rate.
+
+When you click on the back of your card, you see the same information that's shown in the front as well as the headline from your OpenSauced profile, amount of time you've been using our product, and your [PR velocity rate](../../docs/community/pr-velocity.md).
+[screenshot of back of the dev card]().
+
+Whether you are a person who's just beginning their open source journey or a maintainer looking for more contributors and/or sponsors, the dev card is a great way to show your audience how active you are in the community.
```

Comment:
Maybe add something like this:

The Dev Card is a shareable snapshot of your contributions, skills, and engagement in the open source community.